src/nbl/video/device_capabilities/device_limits.json

```diff
@@ -992,29 +988,34 @@
                 {
                     "type": "bool",
                     "name": "shaderUniformBufferArrayNonUniformIndexingNative",
-                    "value": false
+                    "value": true,
+                    "expose": "REQUIRE"
                 },
                 {
                     "type": "bool",
                     "name": "shaderSampledImageArrayNonUniformIndexingNative",
-                    "value": false,
+                    "value": true,
+                    "expose": "REQUIRE",
                     "comment": ["promotion candidate"]
                 },
                 {
                     "type": "bool",
                     "name": "shaderStorageBufferArrayNonUniformIndexingNative",
-                    "value": false
+                    "value": true,
+                    "expose": "REQUIRE"
                 },
                 {
                     "type": "bool",
                     "name": "shaderStorageImageArrayNonUniformIndexingNative",
-                    "value": false,
+                    "value": true,
+                    "expose": "REQUIRE",
                     "comment": ["promotion candidate"]
                 },
                 {
                     "type": "bool",
                     "name": "shaderInputAttachmentArrayNonUniformIndexingNative",
-                    "value": false,
+                    "value": true,
+                    "expose": "REQUIRE",
                     "comment": ["promotion candidate"]
```

Comment:
if your GPU doesn't support those, then you need to revert this part of the JSON, right?

Reply:
yes done